src/documents/en/host/install/install-on-raspberry.html.md

```diff
@@ -26,58 +25,82 @@ lightweight alternative called
-[Raspbian](http://www.raspbian.org/) distribution.
+In order to run Cozy on your Raspberry Pi 2, the simplest way is to download
+our image and set it up on a SD card. The image is based on the [Raspbian Jessie lite](https://www.raspberrypi.org/downloads/raspbian/).
 
 First, get the image archive:
 
 ```bash
-# Download the image (~ 650MB)
-wget https://files.cozycloud.cc/2015-06-16-cozy-raspberry-pi2.img.7z
+# Download the image (~ 680MB)
+wget https://files.cozycloud.cc/cozy-raspberrypi-latest.zip
 
 # Uncompress it
-7zr e 2015-06-16-cozy-raspberry-pi2.img.7z
+unzip cozy-raspberrypi-latest.zip
 ```
 
-Then, find your SD card reference - we are assuming it is */dev/sdc* here -
-and make a low-level dump of the image onto your card using `dd`.
-
 **WARNING:** This operation will erase all data previously stored on the card.
 
-```bash
-sudo dd bs=4M if=2015-06-16-cozy-raspberry-pi2.img of=/dev/sdc
+Follow the guide [Installing operating system images](https://www.raspberrypi.org/documentation/installation/installing-images/README.md).
+
+---
+
+Now, insert the SD card in your Raspberry Pi and boot it.
+
+If you are not able to know your Pi's IP address on the console, you can use
+`nmap` on your local workstation:
+
+```
+nmap -T4 -sP 192.168.x.0/24  # Or your sub network
 ```
 
-On **Windows**, you can use a tool like [Win32 Disk Imager](http://sourceforge.net/projects/win32diskimager/)
-to do the same thing.
+Login to the board via SSH, you can use the default Raspbian credentials
+```pi``` and ```raspberry```:
 
----
+```bash
+$ ssh pi@<ip-of-your-raspberrypi>
 
-Now, insert the SD card in your Raspberry Pi, boot it, and you 
-should be able to access your Cozy through port 443
-(https protocol):
+# Switch as root to type all there commands:
+sudo -s
 
-`https://ip.of.your.pi`
+## Reset CouchDB tokens:
 
-**WARNING:** Installing and configuring Cozy may take quite a long time, one
-hour or so, before the server starts listening on port 443, so be patient.
+# Shutdown the cozy:
+supervisorctl stop cozy-controller
 
-<br>
+# Check CouchDB credientials:
+admin_user="$(sed -n '1,1p' /etc/cozy/couchdb.login)";admin_pass="$(sed -n '2,1p' /etc/cozy/couchdb.login)";curl http://${admin_user}:${admin_pass}@127.0.0.1:5984/cozy
 
-If you are not able to know your Pi's IP address, you can use `nmap`
-to get it on UNIX systems.
+# Delete the CouchDB user:
+admin_user="$(sed -n '1,1p' /etc/cozy/couchdb.login)";admin_pass="$(sed -n '2,1p' /etc/cozy/couchdb.login)";curl -X DELETE http://${admin_user}:${admin_pass}@127.0.0.1:5984/_config/admins/${admin_user}
 
-```
-nmap -T4 -sP 192.168.x.0/24
-```
+# Check if user realy destroy:
+curl http://127.0.0.1:5984/_config/admins/
 
----
+# Need to return: {}
 
-If you want to login to the board via SSH, you can use the default
-Raspbian credentials ```pi``` and ```raspberry```:
-```
-ssh pi@<ip-of-your-raspberrypi>
+# If it look like good, remove file containing credentials informations:
```

Comment:
* "It it looks good"
* "remove <the> file"
* "information" (no "s" even for plural form)